client/my-sites/checkout/upsell-nudge/difm-upsell/index.jsx

```diff
@@ -0,0 +1,166 @@
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Internal dependencies
+ */
+import { CompactCard, Button } from '@automattic/components';
+import DocumentHead from 'calypso/components/data/document-head';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Image dependencies
+ */
+import premiumThemesImage from 'calypso/assets/images/illustrations/themes.svg';
+
+export class DifmUpsell extends PureComponent {
+	render() {
+		const { receiptId, translate } = this.props;
+
+		const title = translate( 'Checkout ‹ Plan Upgrade', {
+			comment: '"Checkout" is the part of the site where a user is preparing to make a purchase.',
+		} );
+
+		return (
+			<>
+				<PageViewTracker path="/checkout/:site/offer-difm/:receipt_id" title={ title } />
+				<DocumentHead title={ title } />
+				{ receiptId ? (
+					<CompactCard className="difm-upsell__card-header">{ this.header() }</CompactCard>
+				) : (
+					''
+				) }
+				<CompactCard className="difm-upsell__card-body">{ this.body() }</CompactCard>
+				<CompactCard className="difm-upsell__card-footer">{ this.footer() }</CompactCard>
+			</>
+		);
+	}
+
+	header() {
+		const { translate } = this.props;
+
+		return (
+			<header className="difm-upsell__small-header">
+				<h2 className="difm-upsell__title">
+					{ translate( "Hold tight, we're getting your site ready." ) }
+				</h2>
+			</header>
+		);
+	}
+
+	body() {
+		const { translate } = this.props;
+		return (
+			<>
+				<h2 className="difm-upsell__header">
+					{ translate(
+						'Skip to the launch line: {{br/}} Let our experts build your dream website',
+						{
+							components: { br: <br /> },
+						}
+					) }
+				</h2>
+
+				<div className="difm-upsell__column-pane">
+					<div className="difm-upsell__column-content">
+						<p>
+							<b>
+								{ translate(
+									'Stand out online with a beautiful, functional, secure website designed and built by WordPress experts just for you.'
+								) }
+							</b>
+						</p>
+						<p>
+							{ translate(
+								'If you’re familiar with WordPress plugins, you know why that’s exciting. Installing plugins on your site is like downloading an app on your phone. Except, instead of getting a new game or productivity tool, you get new features and functionality for your site.'
+							) }
+						</p>
+						<p>
+							{ translate(
+								'There are more than 50,000 WordPress plugins available to help turn your site into any powerful platform you imagine.'
+							) }
+						</p>
+						<p>{ translate( 'Our premium website building service is perfect for:' ) }</p>
+						<ul className="difm-upsell__checklist">
+							<li className="difm-upsell__checklist-item">
+								<Gridicon icon="checkmark" className="difm-upsell__checklist-item-icon" />
+								<span className="difm-upsell__checklist-item-text">
+									{ translate( 'Online stores', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Best for Online Stores:' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/difm-upsell-ab-test)